src/components/custom-field-input-single-choice/custom-field-input-single-choice.test.jsx

```diff
@@ -261,4 +262,79 @@ describe('src/components/custom-field-input-single-choice/custom-field-input-sin
       });
     });
   });
+
+  describe('dropdown close behavior', () => {
+    it('closes the dropdown when clicking outside', async () => {
+      const choices = [{
+        id: 'broke',
+        label: 'broke my heart',
+      }, {
+        id: 'now',
+        label: "now I'm aching for you",
+      }];
+
+      render(
+        <div>
+          <span>CLOSE</span>
+          <CustomFieldInputSingleChoice {...requiredProps} choices={choices} />
+        </div>,
+      );
+
+      userEvent.click(screen.getByLabelText('Test label'));
+      expect(screen.getByText('broke my heart')).toBeInTheDocument();
+      userEvent.click(screen.getByText('CLOSE'));
+      await waitFor(() => expect(screen.queryByText('broke my heart')).not.toBeInTheDocument());
+    });
+
+    it('closes the dropdown when tabbing away', async () => {
+      const choices = [{
+        id: 'broke',
+        label: 'broke my heart',
+      }, {
+        id: 'now',
+        label: "now I'm aching for you",
+      }];
+
+      render(
+        <div>
+          <CustomFieldInputSingleChoice {...requiredProps} choices={choices} />
+          <input />
+        </div>,
+      );
+
+      userEvent.click(screen.getByLabelText('Test label'));
+      expect(screen.getByText('broke my heart')).toBeInTheDocument();
+
+      fireEvent.focus(screen.getByText('broke my heart'));
+      userEvent.tab();
+
+      await waitFor(() => expect(screen.queryByText('broke my heart')).not.toBeInTheDocument());
+    });
+
+    it('resets the inputs state', async () => {
+      const choices = [{
+        id: 'broke',
+        label: 'broke my heart',
+      }, {
+        id: 'now',
+        label: "now I'm aching for you",
+      }];
+
+      render(
+        <div>
+          <span>CLOSE</span>
+          <CustomFieldInputSingleChoice {...requiredProps} choices={choices} />
+        </div>,
+      );
+
+      expect(screen.getByLabelText('Test label')).toHaveValue('');
+      userEvent.click(screen.getByLabelText('Test label'));
+      fireEvent.change(document.activeElement, { target: { value: 'broke my' } });
```

Comment:
```suggestion
      userEvent.type(screen.getByLabelText('Test label'), 'broke my');
```

This does both: click and then types into a value.